src/annotator/anchoring/test/types-test.js

```diff
@@ -142,23 +152,26 @@ describe('annotator/anchoring/types', () => {
 
     describe('#fromRange', () => {
       it('returns a TextPositionAnchor instance', () => {
-        fakePosFromRange.returns({
-          start: 0,
-          end: 1,
-        });
-        const anchor = TextPositionAnchor.fromRange(container, new Range());
-        assert.calledOnce(fakePosFromRange);
-        assert.instanceOf(anchor, TextPositionAnchor);
+        FakeTextRange.instance.relativeTo.returns(FakeTextRange.instance);
+        const range = new Range();
+
+        const anchor = TextPositionAnchor.fromRange(container, range);
+
+        assert.calledWith(FakeTextRange.fromRange, range);
+        assert.calledWith(FakeTextRange.instance.relativeTo, container);
+        assert.equal(anchor.start, FakeTextRange.instance.start.offset);
+        assert.equal(anchor.end, FakeTextRange.instance.end.offset);
       });
     });
 
     describe('#fromSelector', () => {
       it('returns a TextPositionAnchor instance', () => {
         const anchor = TextPositionAnchor.fromSelector(container, {
           start: 0,
-          end: 0,
+          end: 1,
```

Comment:
Is this change an affordance for the test or does it indicate that the operation would get a different answer than it used to? I'm guessing the former but wanted to ask.

Reply:
This change is due to a change to make sure the `start` and `end` values are different. Otherwise the implementation could mix them up and the tests would still pass.